common/djangoapps/student/views/management.py

```diff
@@ -676,7 +676,7 @@ def password_change_request_handler(request):
 
     if email:
         try:
-            from openedx.core.djangoapps.user_api.accounts.api import request_password_change
+            from openedx.core.djangoapps.user_authn.views.password_reset import request_password_change
```

Comment:
Not necessarily as part of this PR, but should the other password-related functions in this file also move to `user_authn`?  For example, `password_change_request_handler` and `password_reset`?  If so, shall we do a deeper assessment of password-related functionality and add to the wiki?